validation/src/main/scala/hmda/validation/dsl/PredicateRegEx.scala

```diff
@@ -26,6 +26,26 @@ object PredicateRegEx {
     override def failure: String = s"is not a valid phone number"
   }
 
+  implicit def validZipCode: Predicate[String] = new Predicate[String] {
+
+    override def validate: (String) => Boolean = {
+      val zipCodeRegex = "^\\d{5}(?:-\\d{4})?$".r
```

Comment:
Are there tests for these regex strings anywhere? It would be good to have proof in the code that they match if and only if we expect them to.